src/jit/gentree.h

```diff
@@ -120,6 +120,8 @@ enum genTreeKinds
     GTK_NOVALUE = 0x0400, // node does not produce a value
     GTK_NOTLIR  = 0x0800, // node is not allowed in LIR
 
+    GTK_NOCONTAIN = 0x1000, // this node is a value, but may not be contained
```

Comment:
> I suspect that will cause some complications due to `GTF_REVERSE_OPS`.

FWIW, `GTF_REVERSE_OPS` isn't terribly meaningful in the backend because tree order is no longer relevant in LIR. We still use this flag to decide in which order we consume the operands for a node, but I don't think that usage is really all that useful and it ought to be easily removed.